api/src/client/ApiFormationHealth.ts

```diff
@@ -41,6 +41,9 @@ const profileData: ProfileData = {
   entityId: "8206864808",
   employerId: "366451721",
   taxId: "752066817169",
+  encryptedTaxId: undefined,
```

Comment:
Not your fault, but I really, really, really hate this pattern. It's yet another instance where we need to manually update a field value.

I desperately want a refactor that generates this for us.